examples/library/Apptainer

```diff
@@ -1,4 +1,5 @@
 Bootstrap: library
+Library: https://library.sylabs.io
```

Comment:
@DrDaveD @gmkurtzer do we need to remove commercial server from our CI tests?

Reply:
Similar to #99, if there is a publicly available service we can leverage, I'd prefer to use that service.

It seems as though Singularity Hub is still up, but I don't recall that it is using the Library API (I think it is using SHUB API), what about any public hosted versions of Singularity Registry?

Reply:
The singularity hub is shut down.  The URL forwards to an archive of some sort, certainly not using Library API.  I agree I would prefer to not use any commercial server for testing, but I don't think there are any.  @vsoch told me there weren't any publicly available sregistry servers.

Reply:
It does not seem that there are any other alternatives to validate the Library API so I think this might be the only option.

Reply:
@gmkurtzer @DrDaveD was there any mention in the new lib-client community (Vanessa, et al) of standing up a publicly available registry?

Did anything happen with artifactory? https://www.jfrog.com/jira/browse/RTFACT-15479?jql=text%20~%20%22singularity%22

Reply:
Most people that use sregistry do so because they want a private registry. It’s easy to use an OCI registry if you just need public.

Reply:
@vsoch The point and question is regarding the Library API, there is no OCI registry that implements that protocol which is why Paul asked if there will be a publicly available implementation of it.

If not, we will probably have to use the Sylabs implementation, or remove the CI testing.

My feeling is we should be testing the Library API, and if there is no non-commercial endpoint for it, then we should use the commercial endpoint, and hope that we don't get ourselves blocked or rate limited (which has much precedent at this point).

Reply:
Yep that makes sense to me! I was just providing further comment on @PaulCharltons's question about a publicly available registry (unlikely to exist).